File: python/001.py
def reverseArr(arr):
    leftPointer,rightPointer = 0,len(arr) - 1

    while leftPointer < rightPointer:
        arr[leftPointer], arr[rightPointer] = arr[rightPointer], arr[leftPointer]
        leftPointer, rightPointer = leftPointer + 1, rightPointer - 1
        
    return arr

print(reverseArr([2,4,5,6,7,8]))

File: golang/001.go
package main

import "fmt"

func revArr(arr []int) []int {
	resArr := []int{}
	for i := len(arr) - 1; i >= 0; i-- {
		resArr = append(resArr, arr[i])
	}
	return resArr
}

func revArrTwoPointer(arr []int) []int {
	leftPointer := 0
	rightPointer := len(arr) - 1

	for leftPointer < rightPointer {
		arr[leftPointer], arr[rightPointer] = arr[rightPointer], arr[leftPointer]
		leftPointer++
		rightPointer--
	}

	return arr
}

func main() {
	fmt.Println(revArrTwoPointer([]int{3, 6, 7, 8, 2, 4}))
}
